pegasus-common/src/main/java/com/linkedin/internal/common/SchemaDirectoryName.java

```diff
@@ -0,0 +1,47 @@
+/*
+   Copyright (c) 2020 LinkedIn Corp.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package com.linkedin.internal.common;
```

Comment:
I am wondering why it is in the internal common package? Users will use it to set up the resolver schema directory name, so it is not for internal using purpose. Do you think we should move this enum to the same package as SchemaResolver?